### PR TITLE
Nerf billy club breaching.

### PR DESCRIPTION
**Pull request by TsunamiAnt**
## About The Pull Request
i ded(or someone else ded, but same thing)

reduces the billy club's structure damage:

Previous everything damage: 48

New Structure(windows, airlock assemblies) damage: 24
New Machinery(airlocks, computers) damage: 30
New Turf(walls) damage: 18

3 people are about as fast with this PR as a single person is right now.

Reinforced walls are now no longer breachable by billy club

## Why It's Good For The Game

because THIS:
image

is NOT good for the game.

## Testing Photographs and Procedure

https://youtu.be/2jONAWjxmFg

## Changelog
:cl:
balance: reduced the billy club's structure damage, you can no longer destroy reinforced walls with it.
/:cl:

### CODE REVIEW

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -975,7 +975,7 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	force = initial(force)
 	armour_penetration = initial(armour_penetration)
 	if(isstructure(A) || ismachinery(A) || isturf(A))
-		force *= 4
+		force *= 2
```
Comment (Rukofamicom) [inline code/game/objects/items/weaponry.dm]: Split the multiplier up:

It should do good bonus damage to airlocks, barricades and such but not to walls. 

We may go back to RCDs being a problem but billy clubs are not the solution - it's finding a way to control RCDs in combat situations or players realizing that it really isn't that hard to make normal walls without one. Making it so attacking the field shorts out the RCD for a while

**Review by Rukofamicom — CHANGES_REQUESTED** [review]

**Comment by RDS88-beeman** [comment]
Considering that, as I recently discovered during the dodgeball round, Billy Clubs are better at taking down reinforced walls than actual tools are, this is a good change.

**Comment by PowerfulBacon** [comment]
This is fine, but I think you should completely remove the extra damage on walls so only breaching doors is effective

**Comment by TsunamiAnt** [comment]
I'll split it up into walls and doors once i get home yeah

**Comment by Geatish** [comment]
In my opinion, there should be a handheld battering ram / breaching hammer for sec to use that spawns in the armory instead of the billy club. Being able to break down any door / electrified window on green is quite unbalanced, since its hard enough as an antag to hide behind doors with an AI onboard.

**Comment by TsunamiAnt** [comment]
I can kinda get that, but like sec has a high and a low. The low is REALLY fucking low, and it's when a perp runs off and there's nothing between you and them except a fucking door and a seemingly deaf ai. that is like one of THE most frustrating things for security.

The high, and i do mean one of the most euphoric highs, is smashing that fucking door or window down while the perp looks on horrified. there is NOTHING better.

So yeah no clue im just gonna split the modifier and leave it at that

**Comment by TsunamiAnt** [comment]
alright new stats:

Previous everything damage: 48

New Structure(windows, airlock assemblies) damage: 24
New Machinery(airlocks, computers) damage: 30
New Turf(walls) damage: 18

I forgot about just removing wall damage at all, so eeeeh this will be fine surely. you'd need like 3 people bashing to get a wall destroyed in the same time as one guy can do it right now. You can still tear down normal walls, but it really is not a viable combat or pursuit strategy. It takes a good long while.

18 is not enough to start scratching reinforced walls at all. They are indestructible for billy clubs.

File: code/game/objects/items/weaponry.dm [hunk 2]
```diff
@@ -974,13 +974,23 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
...
 		// To prevent unnecessary force string calculation (we want this to be treated
 		// as if it wasn't changed)
 		last_force_string_check = force
-	return ..()
+		return ..()
+	else if(ismachinery(A))
+		force *= 2.5
+		armour_penetration += 50
+		last_force_string_check = force
+		return ..()
```
Comment (PowerfulBacon) [inline code/game/objects/items/weaponry.dm]: Why is machinery different to structure?

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]

**Comment by Fronsis** [comment]
Seems good! Could you upload evidence when you have time? I'd like to see the amount of hits it takes for doors/walls/windoors now as a single person compared to current billy clubs

File: code/game/objects/items/weaponry.dm (hunk 2 above)
Comment (TsunamiAnt) [reply code/game/objects/items/weaponry.dm]: Because machinery usually has a bit more health. This felt right in testing!

**Review by Rukofamicom — APPROVED** [review]

**Comment by TsunamiAnt** [comment]
> Seems good! Could you upload evidence when you have time? I'd like to see the amount of hits it takes for doors/walls/windoors now as a single person compared to current billy clubs

https://youtu.be/2jONAWjxmFg

here you go babe. poured all my heart into this

**Comment by Fronsis** [comment]
> > Seems good! Could you upload evidence when you have time? I'd like to see the amount of hits it takes for doors/walls/windoors now as a single person compared to current billy clubs
> 
> https://youtu.be/2jONAWjxmFg
> 
> here you go babe. poured all my heart into this

Holy moly... Thank you for taking the time! 

It seems fair! i assume the regular window would still be easily breachable alongside the Windoor since those are the only two that you missed(and also most frequent to break besides doors and tables) If you work together it should be more than doable in a reasonable time where it doesn't become frustrating, i miight feel that R-windows timer should be maybe 15-14 secs but considering they still retain the capability to deny shocking perhaps it's fine.

One last question! regarding walls, how does it compare against something that a normal crewmember could use(like a toolbox or axe) Since seeing the wall taking 34 hits made me thought that perhaps it got nerfed too much and at the end of the day IF random tool box could be faster to break down a wall but i don't recall their exact number of damage they do.

**Comment by TsunamiAnt** [comment]
I think normal melee weapons do about 12 to 15 damage. The baton does ~20 damage to walls and such

**Review by PowerfulBacon — APPROVED** [review]

**Comment by Rukofamicom** [comment]
> how does it compare against something that a normal crewmember could use(like a toolbox or axe)

It's still better, it just doesn't melt them

**Pull request merged**